kube/kd/deployment.yaml

```diff
@@ -205,33 +205,33 @@ spec:
             - name: DB_HOST
               valueFrom:
                 secretKeyRef:
-                  name: {{.KUBE_NAMESPACE}}-rds
-                  key: endpoint
+                  name: {{.KUBE_NAMESPACE}}-info-rds
+                  key: host
             - name: DB_PORT
               valueFrom:
                 secretKeyRef:
-                  name: {{.KUBE_NAMESPACE}}-rds
+                  name: {{.KUBE_NAMESPACE}}-info-rds
                   key: port
             - name: DB_NAME
               valueFrom:
                 secretKeyRef:
-                  name: {{.KUBE_NAMESPACE}}-rds
-                  key: db_name
+                  name: {{.KUBE_NAMESPACE}}-info-rds
+                  key: name
             - name: DB_SCHEMA_NAME
               valueFrom:
                 secretKeyRef:
-                  name: hocs-info-service
-                  key: rds_schema
+                  name: {{.KUBE_NAMESPACE}}-info-rds
+                  key: schema
             - name: DB_USERNAME
               valueFrom:
                 secretKeyRef:
-                  name: hocs-info-service
-                  key: rds_user
+                  name: {{.KUBE_NAMESPACE}}-info-rds
+                  key: user_name
             - name: DB_PASSWORD
               valueFrom:
                 secretKeyRef:
-                  name: hocs-info-service
-                  key: rds_password
+                  name: {{.KUBE_NAMESPACE}}-info-rds
+                  key: password
```

Comment:
This points to `roots` password?

Reply:
hocs-info-service doesn't contain the root user, the cs-*-rds one does